backend/accounts/account.go

```diff
@@ -52,8 +62,7 @@ type Interface interface {
 	// SendTx signs and sends the active tx proposal, set by TxProposal. Errors if none available.
 	SendTx() error
 	FeeTargets() ([]FeeTarget, FeeTargetCode)
-	TxProposal(string, coin.SendAmount, FeeTargetCode, map[wire.OutPoint]struct{}, []byte, string) (
-		coin.Amount, coin.Amount, coin.Amount, error)
+	TxProposal(*TxProposalArgs) (coin.Amount, coin.Amount, coin.Amount, error)
```

Comment:
This is a very nice change! Now I know what each one means when looking at the `TxProposal` here in the interface. Would be cool if returnd values were also named. Unclear which `coin.Amount` is which.